### DISCUSSION TITLE
s3 support

### DISCUSSION THREAD

**Discussion by svdrummer** [Bodmer/TFT_eSPI#2158] [New processor/display support or capability request]
is there any way to have support for the T-display S3 version, rather than having to use the special library with the same name as this one?

**Comment by Bodmer** [comment]
There are various T display board but maybe this is the one:
https://github.com/Bodmer/TFT_eSPI/discussions/2134

That update is pending but the last patch file is working apparently. I don't have one of those displays so cannot test.

**Comment by Bodmer** [comment]
Continue discussion if required in #2134.
